pkg/agent/controller/networkpolicy/reject_test.go

```diff
@@ -465,6 +470,21 @@ func Test_getRejectPacketOutMutateFunc(t *testing.T) {
 				builder.EXPECT().AddResubmitAction(nil, &l3ForwardingTableID).Return(builder)
 			},
 		},
+		{
+			name: "RejectLocalToRemoteNoFlexibleIPAM,ExternalNode",
+			args: args{
+				rejectType:        RejectPodLocalToRemote,
```

Comment:
Please check the golang job failure, this `RejectPodLocalToRemote` shows undefined.

Reply:
Done.